packages/entities/entities-plugins/src/locales/en.json

```diff
@@ -757,5 +757,62 @@
       "cp_group_label": "Control Plane",
       "realm_group_label": "Realms (Can only select one)"
     }
+  },
+  "rla": {
```

Comment:
Maybe "sp" for `service-protection`? If this might be used in other plugins, we can think of a more generic key name which is not bound to a specific plugin.